server/src/main/java/org/elasticsearch/index/fielddata/FieldDataStats.java

```diff
@@ -47,6 +48,12 @@ public FieldDataStats() {
 
     }
 
+    public FieldDataStats(StreamInput in) throws IOException {
+        memorySize = in.readVLong();
```

Comment:
do you think we could make these fields private? There are getters for them already

Reply:
I pushed ae35d0a8e7db6cb4a5754712fb3dcb07a89e1733, please review again.